opensrp-path/src/main/java/org/smartregister/path/service/intent/ValidateIntentService.java

```diff
@@ -0,0 +1,152 @@
+package org.smartregister.path.service.intent;
+
+import android.app.IntentService;
+import android.content.Context;
+import android.content.Intent;
+import android.util.Log;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.smartregister.domain.Response;
+import org.smartregister.path.R;
+import org.smartregister.path.application.VaccinatorApplication;
+import org.smartregister.repository.BaseRepository;
+import org.smartregister.repository.EventClientRepository;
+import org.smartregister.service.HTTPAgent;
+
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by keyman on 11/10/2017.
+ */
+public class ValidateIntentService extends IntentService {
+
+    private Context context;
+    private HTTPAgent httpAgent;
+    private static final int FETCH_LIMIT = 100;
+    private static final String VALIDATE_SYNC_PATH = "rest/validate/sync";
+
+
+    public ValidateIntentService() {
+        super("ValidateIntentService");
+    }
+
+    @Override
+    public int onStartCommand(Intent intent, int flags, int startId) {
+        context = getBaseContext();
+        httpAgent = VaccinatorApplication.getInstance().context().getHttpAgent();
+        return super.onStartCommand(intent, flags, startId);
+
+    }
+
+    @Override
+    protected void onHandleIntent(Intent intent) {
+
+        try {
+
+             int fetchLimit = FETCH_LIMIT;
+            EventClientRepository db = VaccinatorApplication.getInstance().eventClientRepository();
+
+            List<String> clientIds = db.getUnValidatedClientBaseEntityIds(fetchLimit);
```

Comment:
is the validation status of what's returned here unset?

Reply:
yea, unset or invalid